Tests/TestData/spy_rvi.txt

```diff
@@ -1,380 +1,380 @@
+6/25/2019 01:30 PM,293.7,293.73,290.64,290.76,82028656,72175198.35,-0.07134526195033382,
+6/26/2019 01:30 PM,291.75,292.31,290.345,290.47,51584902,69505797.1,-0.11528836149931332,
+6/27/2019 01:30 PM,291.31,292.06,290.89,291.5,40355239,68397370.05,-0.1491730586795767,
+6/28/2019 01:30 PM,292.58,293.55,292.01,293,59350929,67021777.4,-0.16946732527187935,
+7/1/2019 01:30 PM,296.68,296.92,294.33,295.66,79107474,66155753.5,-0.17572248912832295,
+7/2/2019 01:30 PM,295.61,296.49,294.68,296.43,61504544,65369386.25,-0.1814550623165708,
+7/3/2019 01:30 PM,297.18,298.82,297.02,298.8,40898896,63855845.2,-0.18448711766775655,
+7/5/2019 01:30 PM,297.44,298.64,296.01,298.46,51677326,62968192.45,-0.1686026887518619,
+7/8/2019 01:30 PM,297.01,298.2579,296.22,296.82,45841772,61546671.1,-0.13525392223780955,
+7/9/2019 01:30 PM,295.54,297.52,295.48,297.19,41101279,60561782.4,-0.08798774216569046,
+7/10/2019 01:30 PM,298.37,299.66,297.78,298.61,58595558,60559494.05,-0.01452911877554625,
+7/11/2019 01:30 PM,299.32,299.58,298.2,299.31,50889481,60749152.5,0.08671706517826108,
+7/12/2019 01:30 PM,299.85,300.73,299.51,300.65,40354410,60319615.2,0.1897105666893676,0.04101645288818447
+7/15/2019 01:30 PM,301.13,301.13,300.19,300.75,33910261,59398892.45,0.2486719857780745,0.13116635512296426
+7/16/2019 01:30 PM,300.65,300.88,299.44,299.71,40567964,59467005.2,0.26382098159131323,0.20455052528407644
+7/17/2019 01:30 PM,299.75,299.93,297.74,297.74,44102443,57400386.2,0.23730806512528987,0.24200076109223886
+7/18/2019 01:30 PM,297.19,299.25,296.7,298.83,51701687,56051751.7,0.18123146924719083,0.2386935914097453
+7/19/2019 01:30 PM,300.04,300.07,296.96,297.17,71081238,53777312.75,0.09099569720345568,0.1986492912566217
+7/22/2019 01:30 PM,297.61,298.5,297.04,297.9,43687404,51796209.65,-0.002637334655036688,0.12985417722859105
+7/23/2019 01:30 PM,299.14,300.03,298.22,300.03,44657973,51649971.8,-0.07118005652694373,0.04779468963618085
+7/24/2019 01:30 PM,299.19,301.44,299.09,301.44,47213232,49909200.6,-0.09387153010594568,-0.02508510254440847
+7/25/2019 01:30 PM,300.94,301,299.11,300,55394092,50099660.1,-0.07488866672726177,-0.06793819577467955
+7/26/2019 01:30 PM,300.76,302.23,300.62,302.01,45084077,50336102,-0.05697726259474461,-0.07761295213135054
+7/29/2019 01:30 PM,301.88,301.93,300.85,301.46,38126462,49274878.65,-0.046962290502798286,-0.06742761320879279
+7/30/2019 01:30 PM,299.91,301.17,299.49,300.72,45849045,47611957.2,-0.032138867405693376,-0.05248444005467349
+7/31/2019 01:30 PM,300.99,301.2,295.2,297.43,104245162,49748988.1,-0.01170361823127466,-0.0378138661071671
+8/1/2019 01:30 PM,297.6,300.87,293.96,294.84,142646576,54836372.1,-0.04076756514597638,-0.029235804487118457
+8/2/2019 01:30 PM,293.85,294.12,290.9,292.62,116749695,58089990.55,-0.0927499668038819,-0.038305200160679564
+8/5/2019 01:30 PM,288.09,288.21,281.72,283.82,178745449,64735174.4,-0.15201900237530114,-0.07179294741771539
```

Comment:
Think these changes look culture-specific? Probably some bug with writing them to file, should be culture invariant?

Reply:
Yes, these are culture-specific, but it respects the date/time format of other files used for indicator validation. I think it's simpler to change the file for the scope of this fix.